CoreStubs/ArchiveControl/module.php

```diff
@@ -23,6 +24,25 @@ private function SetVariableData($VariableID, $Data)
         $this->Archive[$VariableID] = $Data;
     }
 
+    public function StubsAddAggregatedValues(int $VariableID, int $AggregationLevel, array $AggregationData)
+    {
+        usort($AggregationData, function ($a, $b)
+        {
+            return $a['TimeStamp'] <=> $b['TimeStamp'];
+        });
+
+        if (isset($this->AggregatedArchive[$VariableID][$AggregationLevel])) {
```

Comment:
Wenn du GetVariableData nutzt, wird dies hier wieder "flacher" wie in AddLoggedData